modules/swagger-codegen/src/main/resources/python/api.mustache

```diff
@@ -75,60 +52,48 @@ class {{classname}}(object):
                  returns the request thread.
         """
 
-        all_params = [{{#allParams}}'{{paramName}}'{{#hasMore}}, {{/hasMore}}{{/allParams}}]
-        all_params.append('callback')
-
-        params = locals()
-        for key, val in iteritems(params['kwargs']):
-            if key not in all_params:
-                raise TypeError(
-                    "Got an unexpected keyword argument '%s'"
-                    " to method {{nickname}}" % key
-                )
-            params[key] = val
-        del params['kwargs']
-
-{{#allParams}}
-{{#required}}
-        # verify the required parameter '{{paramName}}' is set
-        if ('{{paramName}}' not in params) or (params['{{paramName}}'] is None):
-            raise ValueError("Missing the required parameter `{{paramName}}` when calling `{{nickname}}`")
-{{/required}}
-{{/allParams}}
-
```

Comment:
`None` is a value, though an empty value, so passing such a value via GET/POST parameters would look like passing an empty string on a server side. Thus, if there is a check for None, then there should be also a check for an empty string.